tests/tck/features/go/GoYieldVertexEdge.feature

```diff
@@ -0,0 +1,1753 @@
+      """
+    Then the result should be, in any order, with relax comparison:
+      | $^.player.name | serve.start_year | serve.end_year | $$.team.name | e                                                                      | props                              |
+      | "Boris Diaw"   | 2003             | 2005           | "Hawks"      | [:serve "Boris Diaw"->"Hawks" @0 {end_year: 2005, start_year: 2003}]   | {end_year: 2005, start_year: 2003} |
+      | "Boris Diaw"   | 2008             | 2012           | "Hornets"    | [:serve "Boris Diaw"->"Hornets" @0 {end_year: 2012, start_year: 2008}] | {end_year: 2012, start_year: 2008} |
+      | "Boris Diaw"   | 2016             | 2017           | "Jazz"       | [:serve "Boris Diaw"->"Jazz" @0 {end_year: 2017, start_year: 2016}]    | {end_year: 2017, start_year: 2016} |
+      | "Boris Diaw"   | 2012             | 2016           | "Spurs"      | [:serve "Boris Diaw"->"Spurs" @0 {end_year: 2016, start_year: 2012}]   | {end_year: 2016, start_year: 2012} |
+      | "Boris Diaw"   | 2005             | 2008           | "Suns"       | [:serve "Boris Diaw"->"Suns" @0 {end_year: 2008, start_year: 2005}]    | {end_year: 2008, start_year: 2005} |
+    When executing query:
+      """
+      GO FROM "Rajon Rondo" OVER serve WHERE serve.start_year >= 2013 AND serve.end_year <= 2018
+      YIELD $^.player.name, serve.start_year, serve.end_year, $$.team.name, rank(edge) as rank, dst(edge) as dst
+      """
+    Then the result should be, in any order, with relax comparison:
+      | $^.player.name | serve.start_year | serve.end_year | $$.team.name | rank | dst         |
+      | "Rajon Rondo"  | 2014             | 2015           | "Mavericks"  | 0    | "Mavericks" |
+      | "Rajon Rondo"  | 2015             | 2016           | "Kings"      | 0    | "Kings"     |
+      | "Rajon Rondo"  | 2016             | 2017           | "Bulls"      | 0    | "Bulls"     |
+      | "Rajon Rondo"  | 2017             | 2018           | "Pelicans"   | 0    | "Pelicans"  |
+    When executing query:
+      """
+      GO FROM "Boris Diaw" OVER * YIELD dst(edge) as id |
+      GO FROM $-.id OVER like YIELD dst(edge) as id |
+      GO FROM $-.id OVER serve YIELD $$ as dstnode
+      """
+    Then the result should be, in any order, with relax comparison:
+      | dstnode                                        |
+      | ("Hornets" :team{name: "Hornets"})             |
+      | ("Spurs" :team{name: "Spurs"})                 |
+      | ("Spurs" :team{name: "Spurs"})                 |
+      | ("Trail Blazers" :team{name: "Trail Blazers"}) |
+      | ("Spurs" :team{name: "Spurs"})                 |
+      | ("Spurs" :team{name: "Spurs"})                 |
+      | ("Spurs" :team{name: "Spurs"})                 |
+    When executing query:
+      """
+      GO FROM 'Thunders' OVER serve REVERSELY YIELD $^ as src,tags($^), id($^), id($$), $$ as dst
```

Comment:
Maybe we should not allow users to use non-alias vertex or edge expression in yield clause.

and should add following test case:
```
GO ... yield id($$) | yield $-.id($$)
```

Reply:
parser report error